lib/apiservers/engine/backends/backends.go

```diff
@@ -532,7 +544,7 @@ func newSession(ctx context.Context, config *config.VirtualContainerHostConfigSp
 		User:       url.UserPassword(config.Username, config.Token),
 		Thumbprint: config.TargetThumbprint,
 		Keepalive:  defaultSessionKeepAlive,
-		UserAgent:  version.UserAgent("vic-engine"),
+		UserAgent:  version.UserAgent("vic-dynamic-config"),
```

Comment:
At least this portion of this PR should be included in any cherry-pick of #7887.